### PR TITLE
add navscan

### PR DESCRIPTION
**Pull request by PetervDooren**
merge after https://github.com/tue-robotics/depthimage_to_navscan/pull/6

### CODE REVIEW

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Might want to rename node names to NOT include implementation. Also right now we don't run this in the simulator. As it is not needed and it might be relatively heavy. So maybe adding an if condition on ROBOT_REAL `if="$(optenv ROBOT_REAL false)"`.

File: launch/navigation/navigation.launch [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     <!-- Get the machine file -->
     <arg name="machine" default="localhost"/>
+    <arg name="robot_real" default="$(optenv ROBOT_REAL false)" />
```
Comment (MatthijsBurgh) [inline launch/navigation/navigation.launch]: 👍🏻 , please also apply to gmapping launch file.

**Review by MatthijsBurgh — COMMENTED** [review]
I didn't consider it to be put as an argument. So good thinking. Now people can overrule the behaviour of the launch file when they want to.

Now we do it this way, we might want to rename, so it is an argument only to enable/disable the navscan conversion.

**Comment by PetervDooren** [comment]
> I didn't consider it to be put as an argument. So good thinking. Now people can overrule the behaviour of the launch file when they want to.
> 
> Now we do it this way, we might want to rename, so it is an argument only to enable/disable the navscan conversion.

I think we should aim to keep it simple, let the user only set robot real as parameter, otherwise they are free to run the node separately.

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**